docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -80,10 +80,10 @@ Q4. What does the message "Input image of shape is larger than mean image from f
 
 .. _question-5:
 
-Q5. What does the message "Mean file is empty" mean?
+Q5. 
```

Comment:
Propose to say: `Deprecated question`. Also, if posssible, let us delete `**A:**` and `**Q5**` - don't think it is needed

Reply:
Agreed, i think we can even write: `This MO argument/option was deprecated and removed, question is not actual anymore`

Reply:
Removed **A** **Q**. Added `This MO argument/option was deprecated and removed, question is not actual anymore`.

Also removed questions 4 and 6 which also refer to deprecated mean_file and this questions are never referred in mo.

Reply:
generated HTML doesn't look nice
![image](https://user-images.githubusercontent.com/5703039/234569809-a49dba17-9bdd-48e0-a30f-9518dd7a7207.png)

i will remove completely the item keeping numbers for other questions the same

Reply:
Done. Now the generated DOCs look the following 
![image](https://user-images.githubusercontent.com/5703039/234819770-28d72fc9-c19c-40b0-abf2-87b6f5cdf68e.png)